### PR TITLE
Fix/a11y updates

### PR DESCRIPTION
**Pull request by ShaunBaker**

### CODE REVIEW

File: src/components/views/close-contact-alert.tsx [hunk 1]
```diff
@@ -40,7 +40,7 @@ export const CloseContactAlert: FC = () => {
           new Date(Number(contacts[0].exposureAlertDate)),
           contacts[0].daysSinceLastExposure
         );
-        setCloseContactDate(format(exposureDate, 'MMMM dd, yyyy'));
+        setCloseContactDate(format(exposureDate, 'MMMM do, yyyy'));
```
Comment (davidjharding) [inline src/components/views/close-contact-alert.tsx:43]: This is the wrong format. `do` will display `2nd` rather than `2`

**Review by dharding — CHANGES_REQUESTED** [review]

File: src/components/views/close-contact-alert.tsx (hunk 1 above)
Comment (ShaunBaker) [reply src/components/views/close-contact-alert.tsx:43]: That is what is in the design

https://app.zeplin.io/project/5f23ee3d1eb2f9215942942b/screen/5f354870e975187eabbe6cfe

Comment (davidjharding) [reply src/components/views/close-contact-alert.tsx:43]: I cannot see that link. Nonetheless, I do not think we are intending to show the day of the week here (which starts with `0th`). I think you want `Do`

Comment (ShaunBaker) [reply src/components/views/close-contact-alert.tsx:43]: https://date-fns.org/docs/format says `do` is day of month, `Do` is day of year...

Screenshot 2020-09-03 at 16 40 20

Comment (davidjharding) [reply src/components/views/close-contact-alert.tsx:43]: Interesting. I was pretty sure the docs I looked at said something else. Nonetheless, I discussed this with the UX team. We do not want to include the `st` or `nd` stuff on the day of the month. It should be `September 1, 2020`

Comment (davidjharding) [reply src/components/views/close-contact-alert.tsx:43]: Incidentally, this https://date-fns.org/v1.28.5/docs/format reflects what I was saying about the format string. I guess they changed it in some version.

Comment (AlanSl) [reply src/components/views/close-contact-alert.tsx:43]: @dharding Those are version 1.x docs, here are version 2.x docs (which we're using) https://date-fns.org/v2.16.1/docs/format

Comment (AlanSl) [reply src/components/views/close-contact-alert.tsx:43]: <strike>Also I was thinking to be most natural we might do dates like `January 2, 2020` when written and `January 2nd, 2020` when spoken - do you think `January 2, 2020` is more natural when spoken in text-to-voice?</strike> Nevermind, I didn't notice that the discussion was about text not an accessibility label

Comment (ShaunBaker) [reply src/components/views/close-contact-alert.tsx:43]: @dharding @helloantoine can the designs be updated to ensure what we are delivering is what people are looking at...

Comment (ShaunBaker) [reply src/components/views/close-contact-alert.tsx:43]: https://nearform.invisionapp.com/d/main#/console/20287854/428104629/preview

File: src/components/atoms/navbar.tsx [hunk 2]
```diff
@@ -183,7 +185,9 @@ const styles = StyleSheet.create({
   settings: {
     flexDirection: 'column',
     justifyContent: 'flex-start',
-    alignItems: 'center'
+    alignItems: 'center',
+    padding: 12,
+    marginVertical: -12
```
Comment (AlanSl) [inline src/components/atoms/navbar.tsx:190]: Is it in intentional that this brings the icon `12` to the left as well as increasing the hit area? Negative margin right seems to sort that out if not

**Review by AlanSl — APPROVED** [review]
One question then LGTM

If there's a request for a different date format to the designs, I think it should be its own issue then we can be sure to address it cleanly everywhere.

File: src/components/atoms/navbar.tsx (hunk 2 above)
Comment (davidjharding) [reply src/components/atoms/navbar.tsx:190]: If this is to increase the hit target, is there a reason we did not use `hitSlop`?

**Comment by floridemai** [comment]
No longer applies. Will open a separate PR for date format fix.

**Pull request closed without merging**